compiler/rustc_error_codes/src/error_codes/E0633.md

```diff
@@ -1,25 +0,0 @@
-The `unwind` attribute was malformed.
```

Comment:
Update the file like the other errors not emitted by the compiler anymore. It's important to keep track of them!

Reply:
Do you have an example of what a removed error code's description looks like?

Reply:
There are a few ones, for example: https://github.com/rust-lang/rust/blob/master/compiler/rustc_error_codes/src/error_codes/E0619.md.

You can find more of them by using `git grep -n 'emitted'` in `compiler/rustc_error_codes/src/error_codes`.